services/app-web/src/components/Modal/ReviewAndSubmitModal/ReviewAndSubmitModal.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useRef, useEffect } from 'react'
+import {
+    Button,
+    ButtonGroup,
+    Modal,
+    ModalFooter,
+    ModalHeading,
+    ModalRef,
+    ModalProps as UswdsModalProps
+} from '@trussworks/react-uswds';
+import styles from './ReviewAndSubmitModal.module.scss'
+
+interface ReviewAndSubmitModalProps {
+    modalTitle?: string,
+    onSubmit?: () => void,
+    onCancel?: () => void,
+    showModal: boolean,
+}
+
+export type ModalProps = ReviewAndSubmitModalProps & UswdsModalProps
+
+export const ReviewAndSubmitModal = ({
```

Comment:
The goal with the /components folder is to make things as generalizable as possible. This is super close but can you also pass in props for the className and modal heading id? From what I can tell the only things left that make this modal related to just the review and submit page are those two things otherwise its generalizable and could be thrown anywhere in the app.  Let me know if you want to pair.

Basically I think this is close to just being "Modal" not ReviewAndSubmitModal". And a reusable Modal would be great!